### ISSUE TITLE
v75 Upgrade - Fix broken workflows (widget data not loading)

### ISSUE THREAD

**Issue by csmith-he** [archesproject/arches-her#1104]
Workflows are broken - error in the console stating empty dictionary related to widgets is missing

Relates to: https://community.archesproject.org/t/issue-with-workflows-in-7-5/2270

**Comment by aj-he** [comment]
The following code forces the `widgets` component to reload via webpack_require and then the workflows work ok.

This can be used to progress the rest of the upgrade work but we'll need to get to the bottom of why this is an issue.



![Image](https://github.com/archesproject/arches-her/assets/25295526/75b71003-3262-47cb-9a15-db0730c3e02a)

```js
if (Object.keys(widgets).length === 0) {
    console.error("[js/models/card-widget] widgets object is empty");
    if (__webpack_require__){
        try {
            widgets = __webpack_require__('../../arches/arches/app/media/js/widgets.js');
        } catch (error) {
            console.error(error);
        }
    }
}
```

**Comment by aj-he** [comment]
In discussion with @chrabyrd a workaround has been added to fix an issue with widgets not loading correctly into the workflows. This will mean that we need to keep this under review as the workflow work progresses.

https://community.archesproject.org/t/issue-with-workflows-in-7-5/2270

**Comment by aj-he** [comment]
fixed in #1122.

individual workflow fixes to follow
